### PR TITLE
Add linters

### PR DESCRIPTION
**Pull request by divinecharlotte**
Hello
In this branch I added:
_style lint and Web hint .
_index.html file including one h tag
_style.css file to add color to the text

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1 +1,33 @@
...
+
+### Usage 
+use linters to check the errors in your html and style 
+
+
+
+
+
+## Authors
+
```
Comment (aimemalaika) [inline README.md]: - [ ] consider adding more description to your readme. like social media links, project screenshots. Consider using the [Microverse readme template](https://github.com/microverseinc/readme-template).

**Review by aimemalaika — COMMENTED** [review]
Hi @divinecharlotte , 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

the-simpsons-homer-simpson

## Highlights
 - All linters passed ✅ 
 - meaningful commit message ✅ 

## Required Changes ♻️

- [ ]  Kindly add a descriptive description on your pull request, like what all you have worked with to this branch.

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification.

**_Please, do not open a new Pull Request for re-reviews. You should use the same Pull Request submitted for the first review, either valid or invalid unless it is requested otherwise._**

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Comment by divinecharlotte** [comment]
already fixed the changes required
thanks.

**Review by prolajumokeoni — APPROVED** [review]
Hi @divinecharlotte, 

Your project is complete!  There is nothing else to say other than... it's time to merge it :shipit: 
Congratulations! 🎉 


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.
Please, remember to tag me in your question so I can receive the notification @prolajumokeoni.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561)  you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**